### ISSUE TITLE
Widgets are broken again :< [1.9.6 (W)]

### ISSUE THREAD

**Issue by b2loo** [PranshulGG/WeatherMaster#213]
Widgets are now too big again, they cannot be sized down from the bigger 3x2. on 1.9.5(w) I could size the Current Pill Down to 2x1 and it fit perfect on my homescreen.

**Comment by PranshulGG** [comment]
can i get a screenshot?

**Comment by b2loo** [comment]
https://github.com/user-attachments/assets/89ad01b7-e0e0-49da-9da9-0dd17c10e2ba

I can also downgrade and show you the expected behavior, if needed.

**Comment by PranshulGG** [comment]
Ok the pill widget can't be fixed

**Comment by b2loo** [comment]
https://github.com/user-attachments/assets/7d68b0c5-e4b0-4e0c-9c79-0195e58b7a1c

Landscape and square both seem to only resize their width.
Though I assume this is intended?

**Comment by PranshulGG** [comment]
Yep, the height is fixed to prevent the elements inside from breaking.

**Comment by b2loo** [comment]
> Yep, the height is fixed to prevent the elements inside from breaking.

is it possible to reduce the vertical size of the Landscape widget to allow it to fit in 1 row? It's would be nice to fit it for the actual size of the rendered object.

**Comment by PranshulGG** [comment]
I could make a different variant for vertical layouts since devices have different resolutions, column setups, and placement options. I just don’t have a clear idea yet on how to handle that.

**Comment by cirnoe** [comment]
I think the widgets should completely fill the widget area to be consistent with other tile widgets. Right now, the widgets leave out some space, which makes them look bad next to tile widgets from other apps:
![Screenshot_20241210-030928_Trebuchet](https://github.com/user-attachments/assets/70d004f8-84ba-427f-9f71-14826f419c89)

Here you can see how the widget does not correspondent the widget area borders:
![Screenshot_20241210-032248_Trebuchet](https://github.com/user-attachments/assets/35ada171-fbc6-4602-8802-7c6cad880391)

**Comment by PranshulGG** [comment]
@cirnoe Yeah, the widget's height is fixed, which is why it leaves some space. I'll make sure to fix this in the next version.
